nss-day-cohort-51/nutshell-vampire-brood
# Mw feature/nutshell tasks 21

# Description: Issue #21 **Tasks** Check box completes a task

## Type of change


# Testing Instructions
Click the checkbox on any Task Card in the Task List. In the DevTools Components, observe the property **completeStatus** change from false to true. The Task will also disappear from the Task list because the Task list only displays uncompleted Tasks.
Here are the tasks with the appropriate properties. Use these by adding them your DB file:
`{
      "id": 1,
      "name": "Backup outlook by the end of day",
      "userId": 1,
      "description": "any text here. just get it down, guy.",
      "dueDate": "2021-10-11",
      "completeStatus": true
    },
    {
      "name": "File documents",
      "userId": 2,
      "description": "any text here. just get it down, guy.",
      "dueDate": "2021-10-15",
      "completeStatus": false,
      "id": 2
    },
    {
      "name": "Review and approve office inspection",
      "userId": 1,
      "description": "any text here. just get it down, guy.",
      "dueDate": "2021-10-15",
      "completeStatus": true,
      "id": 3
    },
    {
      "id": 4,
      "name": "Be in the middle of a Panhandle!",
      "userId": 1,
      "description": "any text here. just get it down, guy.",
      "dueDate": "2021-10-12",
      "completeStatus": false
    },
    {
      "name": "File Chipmunk Bro's Complaint",
      "userId": 1,
      "description": "Remember to stop and say what's up to Chipmunk Bro",
      "dueDate": "2021-10-11",
      "completeStatus": false,
      "id": 5
    },
    {
      "name": "Pickup the groceries",
      "userId": 1,
      "description": "Bingo! You nailed it!",
      "dueDate": "2021-10-15",
      "completeStatus": false,
      "id": 6
    },
    {
      "name": "Pickup retainers",
      "userId": 1,
      "description": "You have a 1600 appointment. Go early! ",
      "dueDate": "2021-10-13",
      "completeStatus": false,
      "id": 7
    },
    {
      "name": "Dad's Birthday Send Text at 10:13",
      "userId": 1,
      "description": "Send Dad a text message at 1013 on 10/13.",
      "dueDate": "2021-10-13",
      "completeStatus": false,
      "id": 8
    },
    {
      "name": "Peanut Butter Jelly Time",
      "userId": 1,
      "description": "Where he at? Where he at?\nWhere he at? Where he at?\nThere he go! There he go!\nThere he go! There he go!",
      "dueDate": "2021-10-15",
      "completeStatus": false,
      "id": 9
    }`

# Checklist:
